less/pages/homepage.less

```diff
@@ -0,0 +1,130 @@
+/*!
+ * Copyright (c) 2021 Eclipse Foundation, Inc.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * Contributors:
+ *   Martin Lowe <martin.lowe@eclipse-foundation.org>
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+*/
+
+section.active-events {
+  & {
+    margin-top: 40px;
+    margin-bottom: 0;
+  }
+  table {
+    font-size: 16px;
+  }
+  table tbody tr td {
+    padding: 16px 8px 16px 0;
+  }
+  @media (min-width: 992px) {
```

Comment:
Can you put this media query to the top with the other &.
I think it would be easier to read.